python/src/aac/plugins/validators/unused_definitions/_validate_used_definitions.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from aac.lang.definitions.definition import Definition
+from aac.lang.definitions.references import get_definition_type_references_from_list
+from aac.lang.language_context import LanguageContext
+from aac.plugins.validators import ValidatorFindings, ValidatorResult
+
+
+PLUGIN_NAME = "Validate definition references"
+VALIDATION_NAME = "Definition references exist"
```

Comment:
I hate naming validations as it's a bit difficult to be succinct and clear. 😬 
A name like "Definition is referenced" may be a little clearer, but this is fairly subjective and it may be beneficial for the team to brainstorm a bit on this and if there is another alternative that's preferable.

Reply:
Went with changing things over to "Definition is referenced". I used that here as well as in the validator yaml, and updated the reference in spec.

That should cover the rename, right?

Reply:
Yup! You've got it.